### DISCUSSION TITLE
Welcome message

### DISCUSSION THREAD

**Discussion by PrimaMateria** [jackMort/ChatGPT.nvim#23] [Ideas]
Hi, thanks for the awesome plugin! You chose very nice welcome art and quote, but it would be nice if it would be possible to turn it off or to specify your own. If you agree, I might as well try to prepare a PR for this.

**Comment by jackMort** [comment]
Hi, It's configurable you can set:
```lua
  welcome_message = "YOUR WELCOME SCREEN..."
```
